src/gtclang/Frontend/GTClangPreprocessorAction.cpp

```diff
@@ -282,9 +284,59 @@ class GTClangLexer {
 
         consumeTokens(peekedTokens);
       }
+
+      // Check for var ARG1 [ = RHS];
+      if(token_.is(tok::identifier) && token_.getIdentifierInfo()->getName() == "var") {
```

Comment:
are we testing this in any kind of example?

Reply:
yes, the example is in  test/integration-test/PassPreprocessor/VarDeclarationsTest.cpp